config/voipmonitor.conf

```diff
@@ -347,10 +350,10 @@ mysqldb = voipmonitor
 #convert_dlt_sll2en10 = no
 
 # To listen on all interfaces:
-#interface = any
```

Comment:
Is GH lying to me? How come this shows as commented out on master, but https://github.com/dialpad/voipmonitor/blob/master/config/voipmonitor.conf#L349 shows something different?